zproject/urls.py

```diff
@@ -173,6 +173,8 @@
     url(r'^users$', rest_dispatch,
         {'GET': 'zerver.views.users.get_members_backend',
          'PUT': 'zerver.views.users.create_user_backend'}),
+    url(r'^users/myprofile$', rest_dispatch,
+        {'GET': 'zerver.views.users.get_user_profile'}),
```

Comment:
ya, but that url has been taken by `zerver.views.pointer.get_profile_backend`, is I need to expose these informations in `zerver.views.pointer.get_profile_backend` or I need to change the url to something like `/users/me/myprofile` ?